main.cpp

```diff
@@ -278,12 +278,13 @@ int main (int argc, char** argv)
 				std::cerr << "Error dispatching: " << retval << std::endl;
 			else if (retval != 0)
 				packets_read = true;
+			else
```

Comment:
We could delete the assignment of curtime in `process_tcp` and `process_udp` and remove the `else` but it needs to be tested to ensure the bandwith calculation does not become wrong